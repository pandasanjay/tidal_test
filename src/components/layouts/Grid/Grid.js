import React from "react"
import "./Grid.scss"

const Grid = (props) => {
    return <div className={`grid-container`}> {props.children}</div >
}
export default Grid