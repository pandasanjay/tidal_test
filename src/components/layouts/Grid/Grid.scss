@import "../../../config";
.grid-container {
  // display: grid;
  grid-auto-columns: auto;
  grid-gap: 10px;
  background-color: $primary-color;
  padding: 10px;
}
