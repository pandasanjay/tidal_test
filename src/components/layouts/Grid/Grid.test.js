import React from "react";
import { render } from "@testing-library/react";
import Grid from "./Grid";
describe("Grid Layout component ", () => {
  it("should render and check the snapshot", () => {
    const { asFragment } = render(<Grid />);
    expect(asFragment()).toMatchSnapshot();
  });
});
