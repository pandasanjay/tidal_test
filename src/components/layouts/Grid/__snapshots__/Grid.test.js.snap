// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Layout component  should render and check the snapshot 1`] = `
<DocumentFragment>
  <div
    class="grid-container"
  >
     
  </div>
</DocumentFragment>
`;
